From: Carlos Garnacho <carlosg@gnome.org>
Date: Tue, 17 Nov 2020 14:50:07 +0100
Subject: clutter/event: Drop clutter_event_get_device_id()

Drop this odd shortcut, it is now unused.

Part-of: <https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1569>
---
 clutter/clutter/clutter-event.c | 23 -----------------------
 clutter/clutter/clutter-event.h |  2 --
 2 files changed, 25 deletions(-)

diff --git a/clutter/clutter/clutter-event.c b/clutter/clutter/clutter-event.c
index 7de6012..44a6651 100644
--- a/clutter/clutter/clutter-event.c
+++ b/clutter/clutter/clutter-event.c
@@ -1008,29 +1008,6 @@ clutter_event_get_event_sequence (const ClutterEvent *event)
   return NULL;
 }
 
-/**
- * clutter_event_get_device_id:
- * @event: a clutter event 
- *
- * Retrieves the events device id if set.
- *
- * Return value: A unique identifier for the device or -1 if the event has
- *   no specific device set.
- */
-gint
-clutter_event_get_device_id (const ClutterEvent *event)
-{
-  ClutterInputDevice *device = NULL;
-
-  g_return_val_if_fail (event != NULL, CLUTTER_POINTER_DEVICE);
-
-  device = clutter_event_get_device (event);
-  if (device != NULL)
-    return clutter_input_device_get_device_id (device);
-
-  return -1;
-}
-
 /**
  * clutter_event_get_device_type:
  * @event: a #ClutterEvent
diff --git a/clutter/clutter/clutter-event.h b/clutter/clutter/clutter-event.h
index 6948428..789e3fd 100644
--- a/clutter/clutter/clutter-event.h
+++ b/clutter/clutter/clutter-event.h
@@ -699,8 +699,6 @@ void                    clutter_event_set_stage                 (ClutterEvent
 CLUTTER_EXPORT
 ClutterStage *          clutter_event_get_stage                 (const ClutterEvent     *event);
 CLUTTER_EXPORT
-gint                    clutter_event_get_device_id             (const ClutterEvent     *event);
-CLUTTER_EXPORT
 ClutterInputDeviceType  clutter_event_get_device_type           (const ClutterEvent     *event);
 CLUTTER_EXPORT
 void                    clutter_event_set_coords                (ClutterEvent           *event,
